packages/cactus-plugin-bungee-hermes/src/main/typescript/view-creation/view.ts

```diff
@@ -85,4 +132,31 @@ export class View {
     });
     return transactions;
   }
+  public setPrivacyPolicyValue(
+    value:
+      | {
```

Comment:
@eduv09 I recommend extracting the inline type definition into it's own type definition, like an interface called `ISetPrivacyPolicyValue` or similar.

Reply:
+1